index.html

```diff
@@ -215,32 +215,65 @@ <h3>Exposing HTML Features That Do Not Directly Map to Accessibility APIs</h3>
       </p>
     </section>
     <section>
-      <h3>HTML Element Role Mappings</h3>
+      <h3 id="minimum-role">Exposing HTML Features That Require a Minimum Role</h3>
+      <p>
+        A <dfn>minimum role</dfn> is the equivalent WAI-ARIA role an element will map to when an element has no more specific implicit role. This can help ensure that users of assistive technologies get the best possible experience for commonly-used and valid HTML markup where otherwise a role would not be exposed.
```

Comment:
The way the first is worded, it seems to me like there is a concept of a hierarchy of role specificity (more of less specific), when I think really there is just no explicit role/generic and everything else?

It's really a nit, you can take or leave, but I'd write:

```suggestion
        A <dfn>minimum role</dfn> is the equivalent WAI-ARIA role an element will map to when an element does not have a specified or implicit non-generic role. This can help ensure that users of assistive technologies get the best possible experience for commonly-used and valid HTML markup where otherwise a role would not be exposed.
```

Reply:
i'm leaning to leave as is, unless you hae a suggestion on how to indicate the list of conditions that follow are essentially what is clarifying what a more specific role means.

additionally, added a commit to clarify that an explicit role must also not be none, presentation or generic.